### PR TITLE
Contact Page related changes

### PR DESCRIPTION
**Pull request by MAQSiddheshH**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -5904,4 +5904,73 @@ li.h4header {
...
+
+/* 
+    Contact US page changes
+
+*/
+
+/*For IPad */
+@media (min-width:767px) {   
+    #Addresses > .row:first-child > div:first-child{
+        padding-right: 0px;
```
Comment (MAQHatimD) [inline css/style.css]: remove px if it is 0

File: css/style.css [hunk 2]
```diff
@@ -5904,4 +5904,73 @@ li.h4header {
...
     }
+}
+
+/* 
+    Contact US page changes
+
+*/
+
+/*For IPad */
+@media (min-width:767px) {
```
Comment (MAQHatimD) [inline css/style.css]: merge with existing media queries if available

File: css/style.css (hunk 1 above)
Comment (MAQSiddheshH) [reply css/style.css]: removed

File: css/style.css (hunk 2 above)
Comment (MAQSiddheshH) [reply css/style.css]: merged

**Pull request closed without merging**